tests/run-pass/ice-3462.rs

```diff
@@ -0,0 +1,29 @@
+// Copyright 2014-2019 The Rust Project Developers. See the COPYRIGHT
+// file at the top-level directory of this distribution.
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+#![allow(clippy::all)]
```

Comment:
Wouldn't this testcase only make sense if it tried to trigger the lint that caused the ICE?
Right now it would not execute any Clippy lint and it wouldn't have caught the ICE before, either.

Reply:
Ah, got confused on how allow() workes, thanks.